### PR TITLE
Mssw chapter17

### PR DESCRIPTION
**Pull request by Spinodal**

### CODE REVIEW

File: chapter11/mssw/OogaboogooCalendar.pm [hunk 1]
```diff
@@ -3,6 +3,19 @@ package OogaboogooCalendar;
...
+our @EXPORT_OK = qw(
+    number_to_day
+    number_to_month
+);
+
+our %EXPORT_TAGS = (
+    all => [ @EXPORT, @EXPORT_OK ],
+);
+
 use Data::Dumper;
```
Comment (alexdeas) [inline chapter11/mssw/OogaboogooCalendar.pm:19]: Missed opportunity to remove unused dumping.

File: mssw/My-List-Util/lib/My/List/Util.pm [hunk 2]
```diff
@@ -4,9 +4,11 @@ use 5.006;
 use strict;
 use warnings;
 
+use Exporter qw(import);
+
 =head1 NAME
 
-My::List::Util - The great new My::List::Util!
+My::List::Util - A Util for Lists!
```
Comment (alexdeas) [inline mssw/My-List-Util/lib/My/List/Util.pm:11]: Making your boilerplate tests pass!

File: chapter11/mssw/OogaboogooCalendar.pm [hunk 3]
```diff
@@ -3,6 +3,19 @@ package OogaboogooCalendar;
 use strict;
 use warnings;
 
+use Exporter qw(import);
+
+our @EXPORT = qw(
+);
```
Comment (alexdeas) [inline chapter11/mssw/OogaboogooCalendar.pm:9]: Lists are empty by default, so you don't really need the `qw()` here.

File: mssw/My-List-Util/lib/My/List/Util.pm [hunk 4]
```diff
@@ -46,7 +56,9 @@ sub sum {
     return $total;
 }
 
-=head2 function2
+=head2 shuffle
+
+Shuffles things
```
Comment (utopia-2) [inline mssw/My-List-Util/lib/My/List/Util.pm:61]: :+1: level of detail.

File: chapter11/mssw/OogaboogooCalendar.pm (hunk 3 above)
Comment (Spinodal) [inline chapter11/mssw/OogaboogooCalendar.pm:9]: just making future diffs easier if we ever needed to add to `@EXPORT`.

**Pull request merged**